transformations/style_paraphraser/README.md

```diff
@@ -0,0 +1,51 @@
+# Paraphrasing through the use of style transfer
+
+This perturbation provides a range of possible styles of writing, enabling an easy use of style transfer paraphrase models originally introduced in the paper [Reformulating Unsupervised Style Transfer as Paraphrase Generation (2020)](https://arxiv.org/abs/2010.05700) (K. Krishna et al.), published at EMNLP 2020.
+
+Author name: Filip Cornell
+Author email: c.filip.cornell@gmail.com
+Author affiliation: KTH Royal Institute of Technology
+
+## What type of a transformation is this?
+
+This is a paraphraser that transfers between different styles. Several models are enabled and should be usable through this interface. This therefore provides an easy-to-use interface for these large paraphrasing models, giving often high-quality paraphrases.
+
+The current styles of writing supporting are:
+
+- Shakespeare
```

Comment:
Maybe add one-sentence descriptions of each one here?

Reply:
Done!